### ISSUE TITLE
Maybe some issues when calculating uncertainty

### ISSUE THREAD

**Issue by SY-Xuan** [clovaai/rainbow-memory#3]
I think there may be some issues when calculating the uncertainty in your code.
https://github.com/clovaai/rainbow-memory/blob/d0442d49b0a3a2e7d9e49106bb92602a3cc2de3e/methods/finetune.py#L548
`
logit = self.model(x)                
logit = logit.detach().cpu()
`
you use the output of FC layer to calculate the uncertainty.

I think that a softmax is needed here.

**Comment by hwany-j** [comment]
First of all, thank you for your interest in my research. We calculate uncertainty as 

`uncertainty = 1 - max(the number of predicting classes) / # of perturbed images`, 

which means we only use the `argmax(logit)`. Hence, we don't need to use softmax when calculating the uncertainty.

**Comment by SY-Xuan** [comment]
Thank you. I understand.
